ha/core/event_analyzer/watcher/watcher.py

```diff
@@ -30,14 +28,13 @@
 from ha.core.event_analyzer.event_analyzer_exceptions import EventParserException
 from ha.core.event_analyzer.event_analyzer_exceptions import SubscriberException
 
-class Watcher(Thread):
+class Watcher:
     """ Watch message bus to check in coming event. """
 
     def __init__(self, consumer_id: int, message_type: str, consumer_group: str,
                 event_filter: Filter, event_parser: Parser, subscriber: Subscriber):
         """
         Initalize Watcher class to monitor message bus event.
-
         Args:
             id (int): Consumer ID for message bus.
             message_type (str): Message type for getting event.
```

Comment:
please change the argument names on docstring

Reply:
Done